### PR TITLE
Add menu icon class

### PR DESCRIPTION
**Pull request by Martskin**
Adds `spectrum-Menu-itemIcon` classname.

## Description
Adds menu specific icon classname for issue #202 and changes all menu instance with icons.

Screen Shot 2019-10-14 at 1 18 05 PM

### CODE REVIEW

File: components/menu/index.css [hunk 1]
```diff
@@ -99,7 +99,7 @@ governing permissions and limitations under the License.
     }
   }
 
-  .spectrum-Icon {
+  .spectrum-Menu-itemIcon {
     /* Don't get smaller, you're an icon! */
     flex-shrink: 0;
     align-self: flex-start;
```
Comment (lazd) [inline components/menu/index.css]: There's another instance below that needs this change. I would say we should also preserve support for the existing markup so we don't have to make a breaking change.

Comment (Martskin) [reply components/menu/index.css]: Ahh! Good catch. Fixed!

**Comment by lazd** [comment]
This looks good. I'd say it's a `feat` since we're adding a new class in a backwards compatible way.

**Comment by Martskin** [comment]
> This looks good. I'd say it's a `feat` since we're adding a new class in a backwards compatible way.

Thanks @lazd ! Do I need to add a new commit with `feat` to this PR?

**Pull request merged**